sdk/security/keyvault/azkeys/CHANGELOG.md

```diff
@@ -1,13 +1,9 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0-beta.2 (2023-11-08)
 
 ### Features Added
-* Added `HsmPlatform` property to `KeyAttributes`
-
-### Breaking Changes
-
-### Bugs Fixed
+* Added the `HsmPlatform` field to the `KeyAttributes` struct
```

Comment:
This should be `HSMPlatform`

Reply:
Thank you for catching this! Fixed in cc1980b